easybuild/easyconfigs/a/aiohttp/aiohttp-3.8.5-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'PythonBundle'
+
+name = 'aiohttp'
+version = '3.8.5'
+
+homepage = 'https://github.com/aio-libs/aiohttp'
+description = "Asynchronous HTTP client/server framework for asyncio and Python."
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+builddependencies = [
+    ('binutils', '2.39'),
+]
+
+dependencies = [
+    ('Python', '3.10.8'),
+]
+
+use_pip = True
+
+# aioredis and aiosignal do not depend on aiohttp, but are commonly used together and share dependencies
+exts_list = [
+    ('multidict', '6.0.4', {
+        'checksums': ['3666906492efb76453c0e7b97f2cf459b0682e7402c0489a95484965dbc1da49'],
+    }),
+    ('yarl', '1.9.2', {
+        'checksums': ['04ab9d4b9f587c06d801c2abfe9317b77cdf996c65a90d5e84ecc45010823571'],
+    }),
+    ('frozenlist', '1.4.0', {
+        'checksums': ['09163bdf0b2907454042edb19f887c6d33806adc71fbd54afc14908bfdc22251'],
+    }),
+    ('async-timeout', '4.0.2', {
+        'checksums': ['2163e1640ddb52b7a8c80d0a67a08587e5d245cc9c553a74a847056bc2976b15'],
+    }),
+    (name, version, {
+        'checksums': ['b9552ec52cc147dbf1944ac7ac98af7602e51ea2dcd076ed194ca3c0d1c7d0bc'],
+    }),
+    ('aioredis', '2.0.1', {
```

Comment:
I don't think this is needed. Output from
```
module load Python/3.10.8-GCCcore-12.2.0
python3 $EBROOTEASYBUILD/easybuild/scripts/findPythonDeps.py aiohttp==3.8.5
```
is
```
List of dependencies in (likely) install order:
[   ('multidict', '6.0.4'),
    ('yarl', '1.9.2'),
    ('frozenlist', '1.4.0'),
    ('async-timeout', '4.0.2'),
    ('aiosignal', '1.3.1'),
    ('aiohttp', '3.8.5')]
```
Also, shouldn't `aiohttp` be the last item to be installed? It might not matter here, I'm not sure.

Reply:
I'm discussing with other maintainers regarding our policy to cleanup dependencies - my guess is `aioredis` _might_ have been needed for earlier versions of `aiohttp`, but maybe not anymore? Then again, simply removing it might also be risky: if it's an optional dependency, that provides additional functionality, maybe we _do_ want to keep it? I'll await their feedback, so please hold off on any changes

Reply:
hey, just seeing.

my decision was to follow the previous recipe. 

i basically just bumped `aiohttp-3.8.3-GCCcore-11.3.0.eb` and kept the comment that `aioredis` and `aiosignal` are included as they are "usually" used together with aiohttp.